index.js

```diff
@@ -3,6 +3,8 @@ var RSS = require('rss');
 var express = require('express');
 var app = express();
 
+var regex_exclude = process.env.DOCKER_HUB_RSS_REGEX_EXCLUDE || false;
```

Comment:
Could this be passed in as a query param so it's dynamic? I see this overlapping with the include/exclude by name in the `dev` branch: https://github.com/TheConnMan/docker-hub-rss/blob/dev/public/js/app.js#L16.